pytest/tests/adversarial/start_from_genesis.py

```diff
@@ -18,6 +18,9 @@
 nodes = start_cluster(
     2, 1, 2, None,
     [["epoch_length", 100], ["block_producer_kickout_threshold", 80]], {})
+if not doomslug:
+    # we expect inconsistency in store in node 0
```

Comment:
Add some high level background why we expect it.
Over time the assumptions get forgotten.

Reply:
++